### DISCUSSION TITLE
How to stop the execution at a node?

### DISCUSSION THREAD

**Discussion by sgodanas** [nxt-dev/nxt_editor#189] [Q&A] [answered]
I have a chain of connected nodes and even when I raise sys.exit(0), the next nodes in the chain are getting executed. Am I missing something?

Thank you.

**Comment by MichaelAldrich** [comment, marked as answer]
This is definitely a bug. [Here's the fix](https://github.com/nxt-dev/nxt/pull/84) for the next update. In the mean time you can raise any exception.

**Reply by sgodanas** [reply]
Thank you Michael :)

**Reply by sgodanas** [reply]
I patched the file on my end and it works! Thanks again.
